package/kernel/linux/modules/other.mk

```diff
@@ -56,7 +56,8 @@ define KernelPackage/bluetooth
 	$(LINUX_DIR)/net/bluetooth/hidp/hidp.ko \
 	$(LINUX_DIR)/drivers/bluetooth/hci_uart.ko \
 	$(LINUX_DIR)/drivers/bluetooth/btusb.ko \
-	$(LINUX_DIR)/drivers/bluetooth/btintel.ko
+	$(LINUX_DIR)/drivers/bluetooth/btintel.ko \
+	$(LINUX_DIR)/drivers/bluetooth/btmtk.ko@ge6.1
```

Comment:
btmtk added in 5.17: https://github.com/torvalds/linux/commit/8c0d17b6b06c5bef45de7e08c5c3cab8367f6cbc